CollisionOperator/CollisionProcessors/collisionProcessor_inter.f90

```diff
@@ -113,25 +113,35 @@ subroutine collide(self, p, tally, thisCycle, nextCycle)
     class(particleDungeon),intent(inout)     :: thisCycle
     class(particleDungeon),intent(inout)     :: nextCycle
     type(collisionData)                      :: collDat
-    character(100),parameter                 :: Here = ' collide (collisionProcessor.f90)'
+    logical(defBool)                         :: virtual
+    integer(shortInt)                        :: thisCollision
+    character(100),parameter                 :: Here = 'collide (collisionProcessor.f90)'
 
     ! Load material index into data package
     collDat % matIdx = p % matIdx()
 
+    ! Choose collision nuclide and general type (Scatter, Capture or Fission)
+    call self % sampleCollision(p, tally, collDat, thisCycle, nextCycle)
+
+    ! In case of a TMS rejection, set collision as virtual
+    if (collDat % MT == noInteraction) then
+      virtual = .true.
+      thisCollision = 0
+    else
+      virtual = .false.
+      thisCollision = 1
```

Comment:
Pedantic, but this variable isn't very clear to me. I understand reading below that it's for counting collisions. I might propose a more intuitive name like 'isCollision' or 'collisionWorth'.

Reply:
collisionCount? I don't like collisionWorth or isCollision. I actually didn't mind thisCollision

Reply:
addCollision perhaps?

Reply:
collisionIncrement is my fav